src/legacy/core_plugins/kibana/common/field_formats/types/source.js

```diff
@@ -46,7 +45,7 @@ export function createSourceFormat(FieldFormat) {
   }
 
   SourceFormat.prototype._convert = {
-    text: (value) => toJson(value),
+    text: (value) => JSON.stringify(value),
```

Comment:
I can't figure out how to hit this code path.  I can get the html function below, but not this text one.  @flash1293 were you able to at all?

Reply:
@Stacey-Gammon No I wasn't, I thought it's just there for completeness sake to comply to the `FieldFormat` interface.

Reply:
any ideas on how this code path gets triggered @spalger ?  I didn't see other Formatters having a `text` function (at least a couple i looked at).

Reply:
@Stacey-Gammon If `_convert` is just one function in a formatter, it is automatically the text-converter (`src/legacy/ui/field_formats/field_format.js:32`) - most formatters don't overwrite the html special case. You can trigger it by applying the formatter to a field in the index pattern and selecting the field in discover

Reply:
Interesting.  Well, it'd be nice to remove it if it's dead code, but don't want to hold the PR up on this. Nice to see that angular stuff gone!